### PR TITLE
Templatise the drupal config

### PR DESCRIPTION
**Pull request by kierenevans**
Fixes https://github.com/continuouspipe/drupal8-lightning/issues/21

### CODE REVIEW

**Comment by rj-ikos** [comment]
Rather than writing to settings.php, can we instead include an external `settings.cp.php` file which we put the variables in.  Also please uncomment the `settings.local.php` line at the bottom of the settings.php file which will help us with local dev if necessary.

**Comment by kierenevans** [comment]
Sorted :)

File: drupal8-apache/7.0/etc/confd/templates/drupal/settings.php.tmpl [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    'database' => '{{ getenv "DRUPAL_DATABASE_NAME" }}',
+    'username' => '{{ getenv "DRUPAL_DATABASE_USERNAME" }}',
+    'password' => '{{ getenv "DRUPAL_DATABASE_PASSWORD" }}',
+    'prefix' => '{{ getenv "DRUPAL_DATABASE_PREFIX" }}',
+    'host' => '{{ getenv "DRUPAL_DATABASE_HOST" }}',
+    'port' => '{{ getenv "DRUPAL_DATABASE_PORT" }}',
+    'namespace' => 'Drupal\\Core\\Database\\Driver\\mysql',
+    'driver' => 'mysql',
+);
+$config_directories['sync'] = 'sites/default/files/config_{{ getenv "DRUPAL_CONFIG_DIRECTORY_SECRET" }}/sync';
```
Comment (andytson-inviqa) [inline drupal8-apache/7.0/etc/confd/templates/drupal/settings.php.tmpl]: just a thought, as this is a interpretted language, we could use PHP's own `$_ENV`?

Comment (kierenevans) [inline drupal8-apache/7.0/etc/confd/templates/drupal/settings.php.tmpl]: Sorted out and rebased on top of master

**Pull request merged**